src/standalone/Snow/Snow.jl

```diff
@@ -253,27 +248,20 @@ auxiliary_domain_names(::SnowModel) = (
     :surface,
     :surface,
     :surface,
+    :surface,
+    :surface,
 )
 
 
 ClimaLand.name(::SnowModel) = :snow
 
-"""
-    scf(x::FT; α = FT(1e-3))::FT where {FT}
-
-Returns the snow cover fraction, assuming it is a heaviside
-function at 1e-3 meters.
-
-In the future we can play around with other forms.
-"""
-function scf(x::FT; α = FT(1e-4))::FT where {FT}
-    return heaviside(x - α)
-end
-
 function ClimaLand.make_update_aux(model::SnowModel{FT}) where {FT}
     function update_aux!(p, Y, t)
         parameters = model.parameters
+        _ρ_liq = FT(LP.ρ_cloud_liq(parameters.earth_param_set))
 
+        p.snow.κ .= snow_thermal_conductivity(parameters.ρ_snow, parameters)
+        @. p.snow.z .= snow_depth(Y.snow.S, parameters.ρ_snow, _ρ_liq)
```

Comment:
probably don't need .= if you use the @. macro?

Reply:
good catch! thanks